chapters/operatorsandexpressions.tex

```diff
@@ -1292,6 +1292,14 @@ \subsection{Constant Expressions}\doublelabel{constant-expressions}
   \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with constant
   subexpressions as argument (and no parameters defined in the function)
   is a constant expression.
+\item
+  Some function calls are parameter expressions even if the arguments
+  are not:
+
+  \begin{itemize}
+  \item
+    \lstinline!ndims(A)!
+  \end{itemize}
```

Comment:
Wouldn't it make sense to say that ndims is constant? (Like the PR topic says) Even in functions with weird arguments and unknown dimensions... the number of dimensions is always constant, right?

Or is this just copy-pasta?

Reply:
I had forgotten to save after updating the text; thx for detecting it.